src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write code here
+const listEl = document.querySelector('ul');
+const listItems = [...listEl.children];
+
+const convertToNumber = str => {
```

Comment:
This function is named 'convertToNumber', but it returns NaN (which is technically a Number in JavaScript) if the input is not a string. This could be misleading. Consider throwing an error or returning the input unmodified if it's not a string.